mmv1/third_party/terraform/tests/resource_storage_bucket_test.go.erb

```diff
@@ -980,6 +980,85 @@ func TestAccStorageBucket_retentionPolicyLocked(t *testing.T) {
 	})
 }
 
+func TestLabelDiffSuppress(t *testing.T) {
+	cases := map[string]struct {
+		K, Old, New        string
+		ExpectDiffSuppress bool
+	}{
+		"missing goog-dataplex-asset-id": {
+			K:                  "labels.goog-dataplex-asset-id",
+			Old:                "test-bucket",
+			New:                "",
+			ExpectDiffSuppress: true,
+		},
+		"explicit missing goog-dataplex-asset-id": {
+			K:                  "labels.goog-dataplex-asset-id",
+			Old:                "test-bucket",
+			New:                "test-bucket",
```

Comment:
I don't think we need to test when `old == new` as I don't think the DiffSuppressFunc will be triggered on the same value. Maybe change New and Old values to be different and test if the diff will be suppressed (expect not to be suppress)

Reply:
you are right! I've updated the cases so they make more sense! :)